core_lib/src/structure/filemanager.cpp

```diff
@@ -400,6 +337,28 @@ Status FileManager::save(const Object* object, QString sFileName)
     return Status::OK;
 }
 
+Status FileManager::flush(const Object* object)
```

Comment:
Any particular reason we use 'flush'? couldn't it simply be write or writeToDisk?

Reply:
No particular reason. A lot of file system API like to use the word `flush` for flushing content to the disk. For example, [this](https://doc.qt.io/qt-5/qfiledevice.html#flush), [this](https://doc.qt.io/qt-5/qtextstream.html#flush) and [this](https://en.cppreference.com/w/cpp/io/manip/flush).

Reply:
That's true but we don't use that particular word ourselves anywhere, we use 'write' instead.

I know flush is used a lot in c++ but not so much in other programming languages anymore. Whether we use flush or write, is not my issue, I just wish for consistency.

Reply:
I am trying to choose a word which means writing to the working folder but not the final `.pclx`. The meaning of `write()` or `writeToDisk()` is too similar to `save()`

Reply:
Can't we call it writeToWorkingFolder then?